spec/models/favorite_spec.rb

```diff
@@ -1,27 +1,46 @@
 require "rails_helper"
 
 RSpec.describe Favorite do
-  describe "class methods" do
-    it "#total_count" do
-      shelter_1 = Shelter.create(name: "Happy Puppies", address: "55 Street St", city: "Danger Mountain", state: "UT", zip: "80304")
-      pet_1 = Pet.create(image: "image.jpeg", name: "Kunga", approximate_age: "1", sex: "male", shelter_id: shelter_1.id)
-      pet_2 = Pet.create(image: "image.jpeg", name: "Honey Pie", approximate_age: "11", sex: "female", shelter_id: shelter_1.id)
-      favorites = Favorite.new([pet_1, pet_2])
-
-      expect(favorites.total_count).to eq(2)
+  describe "#total_count" do
+    it "can calculate the total number of items it holds" do
```

Comment:
I recognize this and the next test as basically the same tests from the cart lesson, but I'm not sure I understand how they apply to our program. Since we can only add each pet once, should we maybe update it so that we are adding a new key-value pair each time instead of adding to an existing key? Maybe I just don't understand how the favorites hash is working, but I don't see a situation where we would be adding to an existing key rather than creating a new one.